docs/usage.md

```diff
@@ -376,6 +376,22 @@ person.addresses.populator:
 
 There is no new converter but a different populator implementation for this.
 
+### PropertyMappingPopulator & ArrayPropertyMappingPopulator
+
+#### Closure based upon your converter
+
+If you need a Closure which is based upon this Converter (e.g. for PropertyMappingPopulation) you can declare it in this
+way and use it in the `PropertyMappingPopulator` configuration as argument for `$mapper`:
+
+```yaml
+converter.closure.factory:
+  class: \Closure
+  factory: [ '\Closure', 'fromCallable' ]
+  arguments:
+    - '@my.converter'
+    - 'convert'
```

Comment:
Could also be written like this, I think. Is it better? At least it's more consistent with the line above.

```suggestion
  arguments: [ '@my.converter', 'convert' ]
```